### PR TITLE
Workflows in Studio

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**
Main documents for the review in the **studio** folder:
The **Workflows** category:

- workflows
- workflow-properties
- worklfows-user-task
- workflows-general-activities
- workflows-system-actions

Page-related:

- page-editor-widgets-events-section (Sections 2.4-2.7)

Microflow-related:

- microflows (Section 3.4)

Domain Model:

- domain-models (Section 2.1)

Security:

- settings-security

Release Notes:
-9.0-and-above

### CODE REVIEW

**Review by JefHellemansMendix — CHANGES_REQUESTED** [review]

File: content/studio/workflow-properties.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+The **Data** section contains information on what data context the workflow is using. 
+
+![Data Section](attachments/workflow-properties/data.jpg)
+
+**Workflow entity** is an entity that is used as a workflow context. This entity should be of a workflow entity type. For more information, see the [Entities and Their Types](domain-models#entity-types) section in the *Domain Model*.
+
+### 2.3 Display Information Section {#display-info}
+
+The **Display Information** section defines the workflow name and its description that are displayed in the running (published) app.
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/studio/workflow-properties.md:46]: Yes, Cit Devs will start working on workflows using an app template, so they will already have this module.

File: content/studio/workflow-properties.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+| Property             | Description                                                  |
+| -------------------- | ------------------------------------------------------------ |
+| Workflow Name        | **Workflow name** is displayed in the running app. The **Workflow name** can contain attributes of the [Workflow entity](#data), the value of which will be displayed to the end-users. For example, you can select the **FullName** attribute from the **Employee Onboarding** data to display the name of the new employee. |
+| Workflow Description | **Workflow Description** is a description of the workflow displayed in the running app. The **Workflow name** can contain attributes of the [Workflow entity](#data), the value of which will be displayed to the end-users. For example, you can add the **FirstDay** attribute from the **Employee Onboarding** data to display the start date of the new employee and assign them to a training group. |
+
+### 2.4 Permissions Section {#permissions}
+
+**Allowed roles** defines the [user role](settings-security#roles-and-permissions) that can trigger the workflow.
+
+![Permissions Section](attachments/workflow-properties/permissions.jpg)
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/studio/workflow-properties.md:61]: I'll resolve this conversation for now, and will add it as a to-do to check on later.

File: content/studio/workflows-general-activities.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+![General Activities](attachments/workflows/general.jpg)
+
+{{% todo %}}[Retake the screenshot above with the Jump activity.]{{% /todo %}}
+
+## 2 Decision {#decision}
+
+A decision is an element that makes a choice based on a condition and follows one and only one of the outgoing paths. For example, you can use it to follow different processes when a new hire works remotely or when they work from the office:
+
+![Decision Example](attachments/workflows-general-activities/decision-example.jpg)
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/studio/workflows-general-activities.md:20]: Changed

File: content/studio/workflows-user-task.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+tags: ["workflow", "workflows", "user task", "task"]
+---
+
+## 1 Introduction
+
+User task allows you to assign a workflow task to a certain user. 
+
+For example, you can add filters and assign a task to the IT department. When the workflow reaches this user task, the task will appear in their inbox:
+
+![Task Inbox](attachments/workflows-user-task/task-inbox.jpg)
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/studio/workflows-user-task.md:14]: Added a to-do for myself to update it.

File: content/studio/workflows-user-task.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+### 2.3 Task Page Section {#task-page}
+
+**Task Page** is an overview page where the allowed roles can track the progress of the user tasks and view its details, such as comments or attachments:
+
+![Task Page](attachments/workflows-user-task/task-page.jpg)
+
+When you create a task page, you can use a user task-specific page templates: 
+
+![User Task Templates](attachments/workflows-user-task/user-task-templates.jpg)
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/studio/workflows-user-task.md:54]: I've deleted info about page templates as it works like this everywhere in Studio, not just for workflows. You get relevant page templates.

File: content/studio/workflows-user-task.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+| ---------- | ------------------------------------------------------------ |
+| None       | No due date is set.                                          |
+| Duration   | You can set the deadline for the user task with the **Due In** option, which indicates the number of hours, days, or weeks the task is due in. Possible values of the property are the following ones:<br /><ul><li>Hour(s)</li><li>Day(s)</li><li>Week(s)</li> </ul> |
+| Expression | You can set a due date for the user task writing an expression. For example, to set a due date to tomorrow, you can use `addDays([%CurrentDateTime%], 1)`. |
+
+### 2.6 Outcomes Section {#outcomes}
+
+The **Outcomes** property allows you to create new outcomes for the user task. These outcomes can be referred to by other elements, such as buttons. For example, you have a process when you need to approve or reject a request. One button on a [task page](#task-page) can refer to the **Approve** outcome of the user task, while the other one can use the **Reject** outcome:
+
+![Outcomes Section](attachments/workflows-user-task/outcomes.jpg)
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/studio/workflows-user-task.md:87]: Thank you! Since this is a minor thing, I've added it as a to-do later for myself.

**Review by JefHellemansMendix — APPROVED** [review]

**Pull request merged**